tests/robot-tests/.env.example

```diff
@@ -12,4 +12,6 @@ RELEASE_COMPLETE_WAIT=120  # (int) Time to wait for a release to publish
 WAIT_MEDIUM=120  # (int) Variable used throughout tests
 WAIT_SMALL=45 # (int) Variable used throughout tests
 WAIT_LONG=180  # (int) Variable used throughout tests
+TIMEOUT=30 # (int) Variable used throughout tests
```

Comment:
Hey Luke, as per Nick's comment on Friday, can we potentially remove these from the run_tests.py script now, in favour of providing environment variables in the CLI instead if we need to do overrides?

I think the syntax is something like:

`TIMEOUT=60; WAIT_LONG=240; pipenv run...`

Or at least in bash it would be that - I think Nick was gonna give you an example?

Reply:
Discussed after standup: The `.env` file will be the source of truth for these variables